doc/configuration.rst

```diff
@@ -731,3 +731,44 @@ user
   Perform file system operations as the specified user instead of $USER.  Since
   this parameter is not honored by any of the other hdfs clients, you should
   think twice before setting this parameter.
+
+
+Per Task Retry-Policy
+---------------------
+
+Luigi also supports defining retry-policy per task.
+
+.. code-block:: python
+
+    class GenerateWordsFromHdfs(luigi.Task):
+
+       retry_count = 5
+
+        ...
+
+    class GenerateWordsFromRDBM(luigi.Task):
+
+       retry_count = 3
+
+        ...
+
+    class CountLetters(luigi.Task):
+
+        def requires(self):
+            return [GenerateWordsFromHdfs(),GenerateWordsFromRDBM()]
+
+        ...
+
+Supported Configurations
+************************
+The configurations below are also definable in luigi config file. Check :ref:`scheduler-config`
+
++------------------------+-----------+
+| Config                 | Section   |
++========================+===========+
+| retry_count            | scheduler |
++------------------------+-----------+
+| disable_hard_timeout   | scheduler |
++------------------------+-----------+
+| disable_window_seconds | scheduler |
++------------------------+-----------+
```

Comment:
I think we should include a part about what is in retry policy and what they can use per task，what do you think about it?